src/controllers/highlights/HighlightPhrase.cpp

```diff
@@ -14,6 +14,8 @@ QColor HighlightPhrase::FALLBACK_REDEEMED_HIGHLIGHT_COLOR =
     QColor(28, 126, 141, 60);
 QColor HighlightPhrase::FALLBACK_FIRST_MESSAGE_HIGHLIGHT_COLOR =
     QColor(72, 127, 63, 60);
+QColor HighlightPhrase::FALLBACK_ELEVATED_MESSAGE_HIGHLIGHT_COLOR =
+    QColor(255, 174, 66, 60);
```

Comment:
![image](https://user-images.githubusercontent.com/41973452/193191622-b90c1144-6a6b-444f-9c1d-a78f7ca5d8eb.png)
Open to suggestions, just picked the color from the `Priority:Medium` label and lowered the Alpha